packages/vite/src/node/plugins/css.ts

```diff
@@ -1112,7 +1112,7 @@ export async function hoistAtRules(css: string) {
   // CSS @import can only appear at top of the file. We need to hoist all @import
   // to top when multiple files are concatenated.
   // match until semicolon that's not in quotes
-  s.replace(/@import\s*(?:"[^"]*"|'[^']*'|[^;]*).*?;/gm, (match) => {
+  s.replace(/@import\s*(?:url\()?(?:"[^"]*"|'[^']*'|[^;]*).*?;/gm, (match) => {
```

Comment:
What if we keep the same pattern?
```suggestion
  s.replace(/@import\s*(?:url\([^\)]*\)|"[^"]*"|'[^']*'|[^;]*).*?;/gm, (match) => {
```

Reply:
Sorry, just saw your suggestion after my commit https://github.com/vitejs/vite/commit/72f03bcc4bdb0e5677de057ab5832bd6b3fa1813 !

I used the same pattern, but not quite the same way as what you suggest. What do you prefer?

Reply:
I see, I think it is better to have the `url(` ... `)` follow the exact same pattern as the others 🤔
At least to me, it is easier to follow. If you think that both versions are doing the same, then let's go with the one I suggested. Thanks!

Reply:
Here it is https://github.com/vitejs/vite/pull/7718/commits/4fd28101351f87398c584c09146a5f41b5bebed5 !

Reply:
Applied this change in [chore: simplify regex](https://github.com/vitejs/vite/pull/7718/commits/784bcc30d3f6985a11c2dab400b4b52deecab2ec) 👍🏼

Reply:
Oops, sorry, I misread your regexp... Thanks for the fix, I was not available for the last hour!

Reply:
No problem! Thanks again for the quick report and PR! And eager to see Histoire hit the public hopefully soon 👏🏼